### ISSUE TITLE
Wrong number of classes is derived from `label_encoder.vocab_size`

### ISSUE THREAD

**Issue by guanqun-yang** [PetrochukM/PyTorch-NLP#116]
## Behaviors
The following code snippet is directly taken from `README.md` of the this library (see [here](https://github.com/PetrochukM/PyTorch-NLP/blob/master/README.md)). I am expecting the following `n_class` to be equal to 2 (i.e. there are only two classes `[1, 2]`) but 3 is returned.

```python
import itertools

import numpy as np

from torchnlp.datasets import imdb_dataset
from torchnlp.encoders.text import WhitespaceEncoder
from torchnlp.encoders import LabelEncoder

from collections import Counter

sentence_corpus = [record["text"] for record in itertools.chain(train, test)]
label_corpus = [record["sentiment"] for record in itertools.chain(train, test)]

sentence_encoder = WhitespaceEncoder(sentence_corpus)
label_encoder = LabelEncoder(label_corpus)

for record in itertools.chain(train, test):
    record["text"] = sentence_encoder.encode(record["text"])
    record["sentiment"] = label_encoder.encode(record["sentiment"])

print(np.unique([record["sentiment"].item() for record in itertools.chain(train, test)]))
# [1 2]

vocab_size = sentence_encoder.vocab_size
n_class = label_encoder.vocab_size

print(vocab_size, n_class)
# 11402 3
```

## Steps to Reproduce the Problem

Directly run the code snippet after `pip install pip install pytorch-nlp`.

**Comment by qqaatw** [comment]
The one more label you pointed out is `<unk>`, which exists by default when you instantiate LabelEncoder.

https://github.com/PetrochukM/PyTorch-NLP/blob/d7814a297811c9b0dfb285fe0475098b86f3d348/torchnlp/encoders/label_encoder.py#L7

**Comment by nitkannen** [comment]
Does this issue need handling?

**Comment by qqaatw** [comment]
> Does this issue need handling?

Hmm..It's somewhat confusing indeed, maybe we can add proper notes or comments in the documentation to make it clear.

**Comment by nitkannen** [comment]
Cool, any other issue that needs handling? I am looking to contribute

**Comment by qqaatw** [comment]
Kindly explore opened issues and find something in which you are interested to contribute :-)
